docs/operate/security.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: "Security"
+sidebar_position: 6
+description: "Restrict access to Restate services"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Admonition from '@theme/Admonition';
+
+# Security
+
+## Private services
+<CH.Section>
+    When registering an endpoint, every service is by default reachable via HTTP requests to the ingress.
+
+    You can configure a service as `private`, such that you can't invoke it over HTTP, through the [Admin API](focus://1[15:28]) ([docs](/references/admin-api#tag/service/operation/modify_service)):
+
+    ```shell
+    curl -X PATCH localhost:9070/services/MyService \
+        -H 'content-type: application/json' \
+        -d '{"public": false}'
+    ```
+
+    You can revert it back to public with [`{"public": true}`](focus://3).
+
+    <Admonition type="info">
+        Private services can still be invoked by other handlers via the SDK.
+    </Admonition>
+</CH.Section>
+
+## Locking down service access
+Only Restate needs to be able to make requests to your services - requests from
+other services or from the ingress will always go via the Restate runtime. It is
+therefore advisable to ensure that only Restate can reach your service.
+Unrestricted access to the services is dangerous.
```

Comment:
Another benefit is to prevent misconfigurations and registering endpoints in the wrong env if operating multiple Restate instances.